e2e_test/batch/types/timestamptz_utc.slt.part

```diff
@@ -165,3 +168,67 @@ query I
 select count('2022-10-01 12:00:00-08:00'::timestamp with time zone);
 ----
 1
+
+query T
+SELECT make_timestamptz(1973, 07, 15, 08, 15, 55.33);
```

Comment:
We can put these in the `local` folder before they are supported in distributed mode.

https://github.com/risingwavelabs/risingwave/blob/483347725eacbe39a9a60b0817ac145aaf205f80/e2e_test/batch/local_mode.slt#L8

Reply:
Thanks! updated